packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -460,7 +460,7 @@ class _SnackBarState extends State<SnackBar> {
     final Color themeBackgroundColor = isThemeDark
       ? colorScheme.onSurface
       : Color.alphaBlend(colorScheme.onSurface.withOpacity(0.80), colorScheme.surface);
-    final ThemeData inverseTheme = ThemeData(
+    final ThemeData inverseTheme = theme.copyWith(
       brightness: brightness,
```

Comment:
This value came from `theme.brightness`. We're trying to move away from ThemeData dependencies, aside from colorScheme and textTheme; can we use `colorScheme.brightness` instead?

Reply:
done